src/app/fyle/add-edit-expense/add-edit-expense.page.ts

```
@@ -3439,7 +3439,7 @@ export class AddEditExpensePage implements OnInit {
 
   getTxnDate(): Date {
     const formValue = this.getFormValues();
-    return formValue?.dateOfSpend && this.dateService.getUTCDate(new Date(formValue.dateOfSpend));
+    return !!formValue?.dateOfSpend ? this.dateService.getUTCDate(new Date(formValue.dateOfSpend)) : null;
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)